### PR TITLE
Include the assets via a hook into the shortcode template with print …

### PR DESCRIPTION
**Pull request by bordoni**
### 🎫 Ticket

[ET-1269] 

### 🎥 Artifacts 
https://www.loom.com/share/17dd9810830746cd9280b2d1100173b0

[ET-1269]: https://theeventscalendar.atlassian.net/browse/ET-1269?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tickets/Commerce/Hooks.php [hunk 1]
```diff
@@ -701,4 +703,14 @@ public function register_event_compatibility_hooks() {
 
 		add_filter( 'wp_redirect', [ tribe( Compatibility\Events::class ), 'prevent_filter_redirect_canonical' ], 1, 2 );
 	}
+
+	/**
+	 * Includes the Assets to the checkout page.
+	 *
+	 * @since TBD
+	 */
+	public function include_assets_checkout_shortcode() {
```
Comment (juanfra) [inline src/Tickets/Commerce/Hooks.php:712]: Should we maybe move this to the checkout shortcode class?

Comment (juanfra) [reply src/Tickets/Commerce/Hooks.php:712]: It's just a line, but as we do it in other places, I think we may want to do the same.

**Review by juanfra — APPROVED** [review]
Looking good!

File: src/Tickets/Commerce/Hooks.php (hunk 1 above)
Comment (bordoni) [reply src/Tickets/Commerce/Hooks.php:712]: Good point moving.

**Review by juanfra — APPROVED** [review]

**Pull request merged**